packages/cardano-services/test/Metadata/HttpMetadataService.test.ts

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable max-len */
+import { Cardano, ProviderError, ProviderFailure } from '@cardano-sdk/core';
+import { ExtMetadataFormat, createHttpMetadataService } from '../../src/Metadata';
+import { adaPoolsExtMetadataMock, cip6ExtMetadataMock, mainExtMetadataMock } from './mocks';
+import { createGenericMockServer } from '@cardano-sdk/util-dev';
+import { dummyLogger } from 'ts-log';
+import url from 'url';
+
+export const mockPoolExtMetadataServer = createGenericMockServer((handler) => async (req, res) => {
+  const result = handler(req);
+
+  res.setHeader('Content-Type', 'application/json');
+
+  if (result.body) {
+    res.statusCode = result.code || 200;
+    return res.end(JSON.stringify(result.body));
+  }
+
+  const reqUrl = url.parse(req.url!).pathname;
+
+  if (reqUrl === `/${ExtMetadataFormat.AdaPools}`) {
+    return res.end(JSON.stringify(adaPoolsExtMetadataMock));
+  } else if (reqUrl === `/${ExtMetadataFormat.CIP6}`) {
+    return res.end(JSON.stringify(cip6ExtMetadataMock));
+  }
+
+  return res.end(JSON.stringify(adaPoolsExtMetadataMock));
+});
+
+describe('HttpMetadataService', () => {
+  describe('healthy state', () => {
+    let closeMock: () => Promise<void> = jest.fn();
+    let serverUrl = '';
+    const extendedMetadataService = createHttpMetadataService(dummyLogger);
+
+    beforeAll(async () => {
+      ({ closeMock, serverUrl } = await mockPoolExtMetadataServer(() => ({})));
+    });
+
+    afterAll(async () => {
+      await closeMock();
+    });
+
+    it('returns extended metadata in ada pools format when fetch from ext metadata url', async () => {
+      const extMetadata: Cardano.StakePoolMetadata = {
+        ...mainExtMetadataMock(),
+        extended: `${serverUrl}/${ExtMetadataFormat.AdaPools}`
+      };
+      const result = await extendedMetadataService.getStakePoolExtendedMetadata(extMetadata);
+
+      expect(result).not.toBeNull();
+      expect(result).toEqual(adaPoolsExtMetadataMock);
+    });
+
+    it('returns extended metadata in cip-6 format when fetch from ext metadata url', async () => {
```

Comment:
```suggestion
    it('returns CIP-6 format when extDataUrl is present in the metadata', async () => {
```

Reply:
All naming suggestions are applied in f99eb6f441e00d9c6dfd52d59e5a92a7a530317b @rhyslbw